### PR TITLE
Create Tools dashboard

### PR DESCRIPTION
**Pull request by Pe5h4**
This PR should create Tools dashboard, where external links should be displayed as icons/images with external links. Configuration should be passed from site or it can be static. Since it was implemented as a Module, it can be optionaly used in the applications build on top of ASAB-WebUI.

### CODE REVIEW

File: doc/tools.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+```
+
+
+## Setup in config file
+
+To load the configuration, external url's and paths of the image locations must be set.
+
+```
+{
+	"TOOLS":{
```
Comment (ateska) [inline doc/tools.md:31]: "tools":  ...

And the content should be array (name is the attribute within).

**Review by ateska — CHANGES_REQUESTED** [review]

**Comment by ateska** [comment]
I would be cool to see a screenshot too :-)

**Comment by Pe5h4** [comment]
1-tools-link-on-hover
2-tools-module

**Review by ateska — APPROVED** [review]

**Comment by ateska** [comment]
Try to put the icon into the <Button>

**Pull request merged**